docs/azure_arc_jumpstart/azure_arc_k8s/iot_uses_cases/aks/_index.md

```diff
@@ -166,6 +166,13 @@ az iot hub device-identity connection-string show --device-id "EdgeDeviceSim" --
 * Modify the file _/etc/aziot/config.toml_ in the VM that we have deployed using your SSH credentials.
 Once inside the VM, we will modify the /etc/aziot/config.toml file including the following lines at the end of the file with the value of the connection string that we have obtained in the previous step: 
 
+```shell
```

Comment:
Good morning @likamrat!
I have added this line again because when trying to follow the manual again after the review I have seen that if we do not explain it, the user will have doubts because those configuration lines do not come in the default file ... so the user does not could continue and generate incidence! If you want, can we comment on it in a moment?

Reply:
@Manuss20 done